<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>TransformComponentWidget</class>
    <widget class="QWidget" name="TransformComponentWidget">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>293</width>
                <height>213</height>
            </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
                <number>6</number>
            </property>
            <property name="leftMargin">
                <number>8</number>
            </property>
            <property name="topMargin">
                <number>0</number>
            </property>
            <property name="rightMargin">
                <number>8</number>
            </property>
            <property name="bottomMargin">
                <number>8</number>
            </property>
            <item>
                <widget class="QGroupBox" name="groupPosition">
                    <property name="title">
                        <string>Position</string>
                    </property>
                    <layout class="QHBoxLayout" name="layoutPosition">
                        <item>
                            <widget class="QLabel" name="lblPosX">
                                <property name="text">
                                    <string>X</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QDoubleSpinBox" name="posX">
                                <property name="decimals">
                                    <number>3</number>
                                </property>
                                <property name="singleStep">
                                    <double>0.100000000000000</double>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLabel" name="lblPosY">
                                <property name="text">
                                    <string>Y</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QDoubleSpinBox" name="posY">
                                <property name="decimals">
                                    <number>3</number>
                                </property>
                                <property name="singleStep">
                                    <double>0.100000000000000</double>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLabel" name="lblPosZ">
                                <property name="text">
                                    <string>Z</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QDoubleSpinBox" name="posZ">
                                <property name="decimals">
                                    <number>3</number>
                                </property>
                                <property name="singleStep">
                                    <double>0.100000000000000</double>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <widget class="QGroupBox" name="groupRotation">
                    <property name="title">
                        <string>Rotation</string>
                    </property>
                    <layout class="QHBoxLayout" name="layoutRotation">
                        <item>
                            <widget class="QLabel" name="lblRotX">
                                <property name="text">
                                    <string>X</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QDoubleSpinBox" name="rotX">
                                <property name="suffix">
                                    <string>°</string>
                                </property>
                                <property name="decimals">
                                    <number>2</number>
                                </property>
                                <property name="singleStep">
                                    <double>1.000000000000000</double>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLabel" name="lblRotY">
                                <property name="text">
                                    <string>Y</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QDoubleSpinBox" name="rotY">
                                <property name="suffix">
                                    <string>°</string>
                                </property>
                                <property name="decimals">
                                    <number>2</number>
                                </property>
                                <property name="singleStep">
                                    <double>1.000000000000000</double>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLabel" name="lblRotZ">
                                <property name="text">
                                    <string>Z</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QDoubleSpinBox" name="rotZ">
                                <property name="suffix">
                                    <string>°</string>
                                </property>
                                <property name="decimals">
                                    <number>2</number>
                                </property>
                                <property name="singleStep">
                                    <double>1.000000000000000</double>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <widget class="QGroupBox" name="groupScale">
                    <property name="title">
                        <string>Scale</string>
                    </property>
                    <layout class="QHBoxLayout" name="layoutScale">
                        <item>
                            <widget class="QLabel" name="lblSclX">
                                <property name="text">
                                    <string>X</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QDoubleSpinBox" name="sclX">
                                <property name="decimals">
                                    <number>3</number>
                                </property>
                                <property name="minimum">
                                    <double>0.000000000000000</double>
                                </property>
                                <property name="value">
                                    <double>1.000000000000000</double>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLabel" name="lblSclY">
                                <property name="text">
                                    <string>Y</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QDoubleSpinBox" name="sclY">
                                <property name="decimals">
                                    <number>3</number>
                                </property>
                                <property name="minimum">
                                    <double>0.000000000000000</double>
                                </property>
                                <property name="value">
                                    <double>1.000000000000000</double>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLabel" name="lblSclZ">
                                <property name="text">
                                    <string>Z</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QDoubleSpinBox" name="sclZ">
                                <property name="decimals">
                                    <number>3</number>
                                </property>
                                <property name="minimum">
                                    <double>0.000000000000000</double>
                                </property>
                                <property name="value">
                                    <double>1.000000000000000</double>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <spacer name="verticalSpacer">
                    <property name="orientation">
                        <enum>Qt::Orientation::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>0</width>
                            <height>0</height>
                        </size>
                    </property>
                </spacer>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
